app/controllers/organizations/adoptable_pets_controller.rb

```diff
@@ -21,14 +22,14 @@ def show
     @pet = Pet.find(params[:id])
     authorize! @pet, with: Organizations::AdoptablePetPolicy
 
-    if current_user&.adopter_foster_account
+    if current_user
       @adoption_application =
         AdopterApplication.find_by(
-          pet_id: @pet.id,
-          adopter_foster_account_id: current_user.adopter_foster_account.id
+          pet: @pet,
+          form_submission: current_user.person.form_submission
```

Comment:
I was thinking we would need to call `person.form_submission.last` because we would allow for multiple form submissions/sets of answers. But as I think of it, perhaps it makes sense to just keep one set (the latest set) of answers as the 'truth'. There's no point in keeping record of historical answers, right? We currently declare `has_one :form_submission` on `Person` so no need to change that in this case. Just raising here for a thought.

I was originally thinking we would track all form submissions i.e., keep the answers every time a form is submitted for a given person, and just just the last one for Matches. But, I cannot think of a compelling reason to have multiple at this stage. Can you?